runironic.sh

```diff
@@ -45,6 +45,8 @@ if ! iptables -C INPUT -i $INTERFACE -p tcp -m tcp --dport 5050 -j ACCEPT > /dev
     iptables -I INPUT -i $INTERFACE -p tcp -m tcp --dport 5050 -j ACCEPT
 fi
 
+mkdir -p /shared/html
```

Comment:
Unrelated?

Reply:
It doesn’t seem to start without this. Ironic conductor looks for /shared/html/boot.pxe.  On metalkube we create this directory in dev scripts

Reply:
That fix was added to runhttpd.sh in https://github.com/metalkube/metalkube-ironic/commit/7f73fc99a06d7c1e24791dccdb9576d1e801bf80 so it shouldn't be needed here.

Reply:
I would like to not have to run anything but the ironic container for my tests.  With fake-hardware, is there a reason to require httpd? There's no real nodes that are going to be hitting it

Reply:
OK, fine to leave it here then.  Although it shouldn't be needed /shared/html is defiuned as http_root in ironic.conf.